tests/testthat/test-outputs.R

```diff
@@ -207,6 +207,14 @@ test_that("can generate summary report from zip file", {
   expect_true(any(grepl("class=\"logo-naomi\"", content)))
 })
 
+test_that("calibration options used in summary report if present", {
+  zip <- hintr_prepare_spectrum_download(a_hintr_output_calibrated)
+  t <- tempfile(fileext = ".html")
+  generate_output_summary_report(t, zip$path, quiet = TRUE)
+  content <- brio::readLines(t)
+  expect_true(any(grepl("Sex and 5-year age group", content)))
+})
+
```

Comment:
@jeffeaton test here should fail if the subnational age calibration is not printed out in the summary report generated from calibrated results. Did you have something else in mind?

Reply:
It did not fail when the `calibration_options` file extension changed—why not? Would like a test that confirms that the printed calibration options come from the `calibration_options.yml` when we expect it to and from `options.yml` when we expect it to. (A test that would have failed and identified the issue that you are fixing in this PR.)